metron-deployment/roles/metron_kafka_topics/defaults/main.yml

```diff
@@ -21,6 +21,7 @@ topics_to_create:
   - { topic: "bro",         num_partitions: 1, replication_factor: 1, retention_gb: 10 }
   - { topic: "yaf",         num_partitions: 1, replication_factor: 1, retention_gb: 10 }
   - { topic: "snort",       num_partitions: 1, replication_factor: 1, retention_gb: 10 }
+  - { topic: "asa", num_partitions: 1, replication_factor: 1, retention_gb: 10 }
```

Comment:
That makes sense. I was thinking about building out the monit scripts, etc to make this as easy as possible for the user to deploy out-of-the-box, but that's a future PR. Is that something that would be valuable to folks? Either way, I can remove this from the current PR.